.travis.yml

```diff
@@ -4,13 +4,17 @@ node_js:
 cache:
   directories:
   - node_modules
+before_script:
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - ./cc-test-reporter before-build
 script:
 - npm test
 - npm run build
 - npm install coveralls
-- npm test -- -u --coverage
-after_success:
-- coveralls < coverage/lcov.info
+- npm run test -- --coverage --no-cache
```

Comment:
Why do we have `--` and `--no-cache` ?
And not `npm run test --coverage`.

Reply:
To run jest via npm test, we can use the command line arguments by inserting a -- between npm test and the Jest arguments as specified in the [docs](https://deltice.github.io/jest/docs/en/cli.html).  I tried npm run test --coverage and it didn't work as expected.   For --no-cache, jest may give wrong coverage details due to caching. When caching is diabled, we won't run into wrong coverage issues.